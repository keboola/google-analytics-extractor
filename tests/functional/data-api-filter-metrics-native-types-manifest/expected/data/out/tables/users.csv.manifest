{"destination":"in.c-ex-google-analytics-cfg.users","incremental":true,"schema":[{"nullable":false,"primary_key":true,"name":"id","data_type":{"base":{"type":"STRING"}}},{"nullable":false,"primary_key":false,"name":"idProperty","data_type":{"base":{"type":"STRING"}}},{"nullable":false,"primary_key":false,"name":"date","data_type":{"base":{"type":"STRING"}}},{"nullable":false,"primary_key":false,"name":"totalUsers","data_type":{"base":{"type":"STRING"}}},{"nullable":false,"primary_key":false,"name":"itemViews","data_type":{"base":{"type":"STRING"}}}]}