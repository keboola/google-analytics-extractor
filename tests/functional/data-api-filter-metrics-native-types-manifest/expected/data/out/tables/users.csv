"eef4262bb69d17cacaf4d12d68ad8eb53e819cfc","255885884","2023-05-16","3","0"
